doc/src/stdlib/multi-threading.md

```diff
@@ -0,0 +1,47 @@
+
+# Multi-Threading
```

Comment:
`multi-threading.md` should remain part of the base docs for now.

Reply:
Still in base. Original file has been split into three - tasks, distributed and threading, with only distributed moved into stdlib.

Reply:
The path of the file is `doc/src/stdlib/multi-threading.md`

Reply:
That is an existing path for standard library doc entries. Excised docs go into the module path at `stdlib/Distributed/docs/src/`. Contents of  `doc/src/stdlib/parallel.md` have been split into

`doc/src/stdlib/parallel.md` - task API
`doc/src/stdlib/multi-threading.md` - threading API
`stdlib/Distributed/docs/src/index.md` which is copied into `doc/src/stdlib/distributed.md` - in `doc/make.jl` 

This is the way other excised modules have been handling docs.

Reply:
:) my mistake then.